sim/paladin/sor.go

```diff
@@ -103,7 +103,7 @@ func (paladin *Paladin) registerSealOfRighteousness() {
 			ActionID:    core.ActionID{SpellID: rank.proc.spellID},
 			SpellSchool: core.SpellSchoolHoly,
 			DefenseType: core.DefenseTypeMelee,
-			ProcMask:    core.ProcMaskEmpty,
+			ProcMask:    core.ProcMaskMeleeMHSpecial,//changed to ProcMaskMeleeMHSpecial, to fix procs from weapons/oils which do proc from SoR, Idk side effects, does WS proc from SoR?
```

Comment:
What's WS here?

Reply:
Wild Strikes

Reply:
ooh hm. Yeah I don't know personally. I know a lot of effort was put in to try to get SoR to work like in-game but it seems like it's still not functioning correctly if it's supposed to proc oils and weapon procs

Reply:
Ya, with ProcMaskEmpty its not procing Art of War, and weapon procs like "Fist of the Forsaken" which it definitely is on live.  I'm pretty sure I'm getting to many Wild Strike procs with it though in the sim. Its kind of hard to test WS though.